be/src/runtime/fragment_mgr.cpp

```diff
@@ -979,37 +984,13 @@ void FragmentMgr::cancel_query_unlocked(const TUniqueId& query_id,
         return;
     }
 
-    if (ctx->second->enable_pipeline_exec()) {
-        for (auto it : ctx->second->fragment_ids) {
-            // instance_id will not be removed from query_context.instance_ids currently
-            // and it will be removed from fragment_mgr::_pipeline_map only.
-            // so we add this check to avoid too many WARNING log.
-            if (_pipeline_map.contains(it)) {
-                cancel_instance_unlocked(it, reason, state_lock, msg);
-            }
-        }
-    } else {
-        for (auto it : ctx->second->fragment_ids) {
-            cancel_fragment_unlocked(it, reason, state_lock, msg);
-        }
+    for (auto it : ctx->second->fragment_ids) {
```

Comment:
rename fragment_ids to fragment_instance_ids

Reply:
done